src/dex-ui/hooks/dao/useSendProposalTransaction.tsx

```diff
@@ -0,0 +1,80 @@
+import { useMutation, useQueryClient } from "react-query";
+import { TransactionResponse } from "@hashgraph/sdk";
+import { DAOMutations, DAOQueries } from "./types";
+import { DexService } from "@services";
+import { useDexContext } from "@hooks";
+import { isNil } from "ramda";
+import { MemberOperationsType } from "@dex-ui/pages/dao/MemberUpdate/types";
```

Comment:
You can import directly from `@pages` if ** MemberUpdate/types** are exported in the index.ts